rse/procedures/job-descriptions.rst

```diff
@@ -79,11 +79,18 @@ they will probably make a move.
 Beyond
 ------
 
-Further career development is not a part of the Aalto RSE program (yet?),
-but beyond this step the RSE could become similar principal investigator who
-develops projects and applies for funding as a co-PI on significant
-national and international grants.  This would be a horizontal move
-outside of our team.
+Further career development is not a part of the Aalto RSE program
+(yet?), and to be honest it's hard to see an internal advancement at
+Aalto.  But at the same time, this isn't a dead-end internal job.  Especially
+the lower levels take the place of, for example, postdoctoral
```

Comment:
```suggestion
Aalto.  But at the same time, this isn't a dead-end internal job.  Especially
at the lower levels it could take the place of, for example, postdoctoral
```
(do you know how to add the suggestions like this?)